spider/viewers/viewer_refinement.py

```diff
@@ -30,7 +30,7 @@
 import pyworkflow.protocol.params as params
 from pyworkflow.viewer import DESKTOP_TKINTER, WEB_DJANGO
 from pwem.viewers import (EmPlotter, ChimeraClientView, ChimeraView,
-                          EmProtocolViewer, ChimeraAngDist)
+                          EmProtocolViewer, ChimeraAngDist, pwutils)
```

Comment:
that's wrong :)

Reply:
You are right @azazellochg I do not know how pwutils ended in that line.  I remove pwutils form the import line.